zerver/views/home.py

```diff
@@ -198,6 +198,8 @@ def home_real(request):
         save_stacktraces      = settings.SAVE_FRONTEND_STACKTRACES,
         server_inline_image_preview = settings.INLINE_IMAGE_PREVIEW,
         server_inline_url_embed_preview = settings.INLINE_URL_EMBED_PREVIEW,
+        password_min_length = settings.PASSWORD_MIN_LENGTH,
```

Comment:
@r0fls Yeah. The defaults are already set before. It's 0 in development env and 6 in production.